lib/custom_widgets/movie_title.dart

```diff
@@ -11,30 +10,44 @@ extension IterableColors on List<Color> {
 }
 
 extension MarioLetters on List<TextSpan> {
```

Comment:
I'd move these extension classes to the bottom of the file, after the widget, to improve readability of the file.

Reply:
done